apps/services/cms-importer/infra/cms-importer-worker.ts

```diff
@@ -0,0 +1,18 @@
+import { service, ServiceBuilder } from '../../../../infra/src/dsl/dsl'
+
+export const workerSetup = (): ServiceBuilder<'cms-importer-worker'> =>
+  service('cms-importer-worker')
+    .image('services-cms')
+    .namespace('cms-importer')
+    .secrets({
+      CONTENTFUL_MANAGEMENT_ACCESS_TOKEN:
+        '/k8s/contentful-entry-tagger/CONTENTFUL_MANAGEMENT_ACCESS_TOKEN',
+    })
+    .command('node')
+    .args('main.js')
+    .extraAttributes({
+      // Schedule to run daily at 15:00 and midnight.
+      dev: { schedule: '0 15,0 * * *' },
+      staging: { schedule: '0 15,0 * * *' },
+      prod: { schedule: '0 15,0 * * *' },
```

Comment:
Why wait 15 hours, then 9 hours? Why not every 12 hours? Have you considered a more frequent schedule on `dev`, to get more information during work hours?
```suggestion
      // Schedule to run daily at 15:00 and midnight.
      dev: { schedule: '0 */12 * * *' },
      staging: { schedule: '0 0,12 * * *' },
      prod: { schedule: '0 */4 * * *' },
```

Reply:
The stakeholders decision, but more frequent runs on dev would be good

Reply:
Rannís updates the grants on these set times

Reply:
All right. Still, while working on the application on `dev`, I'd recommend a more frequent schedule; at least in a feature-deployment.

Reply:
yeah i agree